.github/workflows/ca-renewal-system-certs-hsm-test.yml

```diff
@@ -271,6 +271,7 @@ jobs:
           docker exec pki pki \
               -u caadmin \
               -w Secret.123 \
+              --ignore-cert-status REVOKED_CERTIFICATE \
```

Comment:
Should this be `EXPIRED_CERTIFICATE` since we're testing with expired certs?

Reply:
This error is generated by checking the validity of temporary `sslserver` certificate. In line 211 above it was verified the expired `sslserver` and the error was `EXPIRED_CERTIFICATE` as expected. Then it is replaced with a temporary certificate which is valid and not revoked but all the subsystem certificates have expired, including the `ocsp_signing` certificate. Therefore, testing the temporary `sslserver` certificate it is valid and good but the validation failed because the OCSP certificate has expired. As a result the certificate is considered REVOKED because our policy is to consider the certificate revoked if validation fails.